roles/ups/templates/role_binding.yml.j2

```diff
@@ -0,0 +1,13 @@
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: unifiedpush-operator
+subjects:
+- kind: ServiceAccount
+  name: unifiedpush-operator
+  namespace: {{ mobile_services_namespace }}
```

Comment:
Are we required to add the namespace metadata in other RoleBindings? Looks like we don't need namespace for RoleBindings @grdryn was there some discussion about this? I recall something but not sure.

Reply:
This is the namespace of the ServiceAccount which is being set as a subject of the ClusterRoleBinding. Since ServiceAccounts are scoped to namespaces, I believe this would always be needed.

The ClusterRoleBinding itself is not a namespaced resource, so we don't specify the namespace. For RoleBindings, I don't think we'd need to specify the namespace in the yaml definition, since we could just specify it with kubectl.